trunk/SUAVE/Components/Energy/Converters/Motor_Lo_Fid.py

```diff
@@ -52,6 +52,8 @@ def __defaults__(self):
         self.gearbox_efficiency = 0.0
         self.expected_current   = 0.0
         self.motor_efficiency   = 0.0
+        self.rated_power        = 0.0
+        self.rated_voltage      = 0.0
     
     def omega(self,conditions):
```

Comment:
Also missing docstrings and doxygen info.

Reply:
Doxygen info added